src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorTableType.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public class ConnectorTableType : FormulaType
```

Comment:
>ConnectorTableType [](http://example.com/codeflow?start=17&length=18)

can we make internal? 

- We will probably want to remove this later. 
- callers shouldn't depend on this distinction (vs just TableType)

Reply:
we need a way to identify this specific type in IR rewriter, either in having a derived TableType or a specific DKind